Makefile

```diff
@@ -0,0 +1,11 @@
+DOCKER_COMPOSE_FILE=_tests/integration/local_docker_test_environment/docker-compose.yml
```

Comment:
I'm generally not a big fan of Makefiles if we also have Bitrise workflows. Would it be possible to add these targets as workflows to `bitrise.yml`?